File: Lib/Telerik/Examples/RichTextEditor/SplashForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace RichTextEditor
{
    public partial class SplashForm : Form
    {
        public SplashForm()
        {
            InitializeComponent();
            this.radWaitingBar1.ThemeName = Program.ThemeName;
        }

        protected override void OnShown(EventArgs e)
        {
            base.OnShown(e);
            this.radWaitingBar1.StartWaiting();
        }

        protected override void OnClosing(CancelEventArgs e)
        {
            base.OnClosing(e);
            this.radWaitingBar1.StopWaiting();
        }

    }
}

File: Lib/Telerik/Examples/RichTextEditor/SplashForm.vb

Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports System

Namespace RichTextEditor
    Partial Public Class SplashForm
        Inherits Form
        Public Sub New()
            InitializeComponent()
            Me.radWaitingBar1.ThemeName = Program.ThemeName
        End Sub

        Protected Overrides Sub OnShown(e As EventArgs)
            MyBase.OnShown(e)
            Me.radWaitingBar1.StartWaiting()
        End Sub

        Protected Overrides Sub OnClosing(e As CancelEventArgs)
            MyBase.OnClosing(e)
            Me.radWaitingBar1.StopWaiting()
        End Sub

    End Class
End Namespace


